backend/src/packages/auth/libs/helpers/token/verify-token/verify-token.helper.ts

```diff
@@ -0,0 +1,10 @@
+import { jwtVerify, type JWTVerifyResult } from 'jose';
```

Comment:
Create a separate token service

Reply:
It was already suggested to Vladyslav in his issue: https://github.com/BinaryStudioAcademy/bsa-2023-calmpal/pull/37#discussion_r1300051916

Should I also do it by myself?

Reply:
So it will prevent of difficult merge conflicts

Reply:
Keep doing, it's cool that you check Vladyslav`s PR and figure out about jwt service)